src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector256.cs

```diff
@@ -2716,6 +3187,40 @@ public static unsafe Vector512<T> ToVector512Unsafe<T>(this Vector256<T> vector)
             return result;
         }
 
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static Vector256<T> Truncate<T>(Vector256<T> vector)
+        {
+            if ((typeof(T) == typeof(byte))
+             || (typeof(T) == typeof(short))
+             || (typeof(T) == typeof(int))
+             || (typeof(T) == typeof(long))
+             || (typeof(T) == typeof(nint))
+             || (typeof(T) == typeof(nuint))
+             || (typeof(T) == typeof(sbyte))
+             || (typeof(T) == typeof(ushort))
+             || (typeof(T) == typeof(uint))
+             || (typeof(T) == typeof(ulong)))
```

Comment:
I'm probably missing it... where is `Truncate<T>` used with any of these types? I see the two public overloads below that call this with float/double.

Reply:
Oh, from the ISimdVector implementation I guess?

Reply:
Right, it's accessible from `ISimdVector` (which is internal only currently) and so this just causes it to no-op.